static/js/modules/profileStats.js

```diff
@@ -0,0 +1,90 @@
+import { serverData } from "./serverData.js";
+
+var profileStatsTable = {
+
```

Comment:
messed up indents

Reply:
Are you referring to the template html? if so, fixed